scripts/learn.py

```diff
@@ -275,6 +277,7 @@ def main():
     # Use this for hyperparams added in a new version, but nonexistent from older versions
     retrocompatibility_defaults = {'feed_previous_direction': False,
                                    'normalize': False,
+                                   'sort_streamlines': False,
```

Comment:
We also need to put this in eval_model.py (or at some point do it clean and have it all in the same place...!)